src/main/java/frc/robot/commands/autos/OneFiveLeave.java

```diff
@@ -23,17 +24,19 @@ public OneFiveLeave(
       Shooter shooter,
       Conveyor conveyor,
       DriveSubsystem drive,
-      MatchState matchState) {
+      MatchState matchState,
+      Command rumbleBrieflyCommand) {
     final double X_DISTANCE_TO_NOTE_METERS = 1.0; // guesstimate
     final double NO_Y_DISTANCE = 0.0;
     final boolean FIELD_RELATIVE = true;
     final double WAIT_BEFORE_DRIVE_SEC = 1.0;
+    Command rumbleCommand = rumbleBrieflyCommand;
 
     addRequirements(intake, elevator, shooter, conveyor, drive);
     addCommands(
         new ScoreThisNote(intake, elevator, shooter, conveyor),
         new ParallelCommandGroup(
-            new IntakeSequence(intake, elevator, conveyor, shooter)
+            new IntakeSequence(intake, elevator, conveyor, shooter, rumbleCommand)
```

Comment:
Maybe make the auto ones a no-op? We don't want to rumble in auto

Reply:
just wondering, what does "no-op" mean?

Reply:
no operation. Pass in an empty command so it doesn't rumble in auto

Reply:
Changed